fluentbit.conf

```diff
@@ -0,0 +1,19 @@
+[SERVICE]
```

Comment:
A more accurate test would be to include the CRIO concat logic in both fluentd and fluentbit https://github.com/openshift/cluster-logging-operator/pull/828/files#diff-eb1e97f55e58e090889a157e0aa8249642ad3a2d3b1845f8bce3d535d50f4541R9 since both will be subject to needing to do this

Reply:
Got it / correct .... there are places for even more improvlments .... for now I think we want to focus on comparison high level with the metrics Pratibha creates and see that the metrics match 

I added TODO remark in the code to remind we need to add this in future versions